vmb/matrimony/models/profiles.py

```diff
@@ -568,6 +568,12 @@ def set_status(self, status_text):
         d = {v: k for k, v in PROFILE_STATUS_CHOICES}
         self.status = d.get(status_text)
 
+    def delete(self, *args, **kwargs):
```

Comment:
First of all, you will need to call `super()` to allow the profile to be deleted. Also, this is a wrong way to do it. What if I do a bulk profile delete on a queryset. That will not even call this `delete` method.

We can achieve the same by setting `on_delete` for the FK definition to `models.CASCADE`.

Reply:
ok